static/js/apps/explore_landing/topics.json

```diff
@@ -262,8 +262,8 @@
             "url": "q=Tell+me+about+health+equity+in+California"
           },
           {
-            "title": "Which communities in Kansas are uninsured",
-            "url": "q=Which+communities+in+Kansas+are+uninsured"
+            "title": "Tell me about health insurance in Kansas",
```

Comment:
Is this intentionally being changed?

Reply:
yes, this was intentional. Guha had asked for it (it was one of those P0s)